Extension/src/LanguageServer/ui.ts

```diff
@@ -57,19 +77,20 @@ export class UI {
     private readonly codeAnalysisTranslationHint: string = "{0} is a program name, such as clang-tidy";
     private runningCodeAnalysisTooltip: string = "";
     private codeAnalysisPausedTooltip: string = "";
+    get isNewUI(): boolean { return false; };
 
     constructor() {
         const configTooltip: string = localize("c.cpp.configuration.tooltip", "C/C++ Configuration");
         this.configStatusBarItem = vscode.window.createStatusBarItem("c.cpp.configuration.tooltip", vscode.StatusBarAlignment.Right, 0);
         this.configStatusBarItem.name = configTooltip;
-        this.configStatusBarItem.command = "C_Cpp.ConfigurationSelect";
+        this.configStatusBarItem.command = "C_Cpp.ConfigurationSelectUI_Telemetry";
```

Comment:
Does "_Telemetry" really need to be in the name of the command itself?  These strings are (potentially) exposed to users.

Reply:
No the names can change, is it an issue to keep them the way they are?